src/StereoDB/StereoDb.fs

```diff
@@ -122,8 +130,44 @@ type internal StereoDb<'TSchema when 'TSchema :> IDbSchema>(logger: ILogger, sch
             with
                 ex -> ()
     }
+
+    let createHeartbeatTopic = task {
+        use adminClient = AdminClientBuilder(AdminClientConfig()).Build()
+        try
+            do! adminClient.CreateTopicsAsync(
+                [TopicSpecification(
+                    Name = _heartbeatTopic
+                )],
+                CreateTopicsOptions()
+            )
+        with 
+        | ex -> logger.Error("Error during creating topic")
```

Comment:
you can remove |
try
    some code
with
    ex -> ()

Reply:
done